cc/position_benchmark.cc

```diff
@@ -1,84 +0,0 @@
-// Copyright 2018 Google LLC
```

Comment:
Was deleting this intentional?

Reply:
Yeah, I pruned a bunch of deps from our WORKSPACE and this was the only benchmark we have. I haven't run it in a year, so there didn't seem much point in keeping it around.